docs/tutorials/getting_started/customize_your_deployment.rst

```diff
@@ -111,16 +111,13 @@ Additional Validation Operators and Actions
 
 Most teams will want to configure various :ref:`Validation Actions` as part of their deployment.
 
-* :ref:`How to re-render Data Docs as a Validation Action`
+* :ref:`how_to_guides__validation__how_to_update_data_docs_as_a_validation_action`
```

Comment:
Just checking, do we want to use the reference tag pattern for all of these going forward, rather than the actual page title? For this PR I think this is fine (no need to clean up everything), but using the tags instead of titles seems much more robust to changes.

Reply:
Yes, I agree we should switch. Let's get rid of all the broken links and warnings first, though.

Reply:
Awesome, just wanted to make sure I understand what's going on. Nice.